<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Contribute</class>
 <widget class="QDialog" name="Contribute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 225, 229);</string>
  </property>
  <widget class="QLineEdit" name="option1LineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="option2LineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="option3LineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="option4LineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="questionTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>631</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="utiiltiyTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>631</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Enter your question, options, correct answer and topic in the boxes above. Your question will be verified and added to database.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitPushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="subjectLabel">
      <property name="text">
       <string>Subject</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="subjectComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>195</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="choiceLabel">
      <property name="text">
       <string>Correct Choice</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="choiceComboBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>questionTextEdit</tabstop>
  <tabstop>option1LineEdit</tabstop>
  <tabstop>option2LineEdit</tabstop>
  <tabstop>option3LineEdit</tabstop>
  <tabstop>option4LineEdit</tabstop>
  <tabstop>choiceComboBox</tabstop>
  <tabstop>subjectComboBox</tabstop>
  <tabstop>submitPushButton</tabstop>
  <tabstop>utiiltiyTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
